### PR TITLE
Limit to use "all" as the name for a language

### PR DESCRIPTION
**Pull request by masatake**
This was one of TODO item for releasing.

### CODE REVIEW

**Comment by masatake** [comment]
This is related to #1711.

**Comment by masatake** [comment]
```
cp ctags.1.rst ctags-optlib.7.rst ctags-incompatibilities.7.rst tags.5.rst  ../docs/man

Files under docs/man/ are not up to date.

Please execute 'make -C man QUICK=1 update-docs' and commit them.

The command "./misc/travis-check.sh" exited with 1.

```
Nice!  I don't have to remember the command line. The error message tells what I have to do.

**Comment by k-takata** [comment]
https://travis-ci.org/universal-ctags/ctags/jobs/609874484#L447
Oh, #2259 caused the issue, sorry.

**Comment by k-takata** [comment]
I created #2308 to fix it.

File: misc/travis-check.sh [hunk 1]
```diff
@@ -60,6 +60,9 @@ if [ "$TARGET" = "Unix" ]; then
 		if ! git diff --quiet docs/man; then
```
Comment (masatake) [inline misc/travis-check.sh]: I think we don't need --quiet option here.
If everything works fine, nothing is printed even if we remove --quiet option.
If something wrong, git diff without --quiet gives us much hint for finding the root cause.

Comment (masatake) [reply misc/travis-check.sh]: I will push a change what I think here though I should make another pull request for the purpose.

**Comment by k-takata** [comment]
I merged #2308. Could you rebase on top of it?

File: misc/travis-check.sh (hunk 1 above)
Comment (k-takata) [reply misc/travis-check.sh]: Without --quiet, git doesn't seem to return the error code even the differences are found.

Comment (masatake) [reply misc/travis-check.sh]: Oh, I see.

Comment (masatake) [reply misc/travis-check.sh]: It looks a bug of git for me:-P.

Comment (masatake) [reply misc/travis-check.sh]:
```
       --exit-code
           Make the program exit with codes similar to diff(1). That is, it exits with 1 if
           there were differences and 0 means no differences.

       --quiet
           Disable all output of the program. Implies --exit-code.
```

So I think we can use --exit-code.

**Pull request merged**